cases/OCP-10674/ruby-ex/Gemfile

```diff
@@ -0,0 +1,5 @@
+sleep 10
+puts "10s delay time passed"
```

Comment:
@wzheng1 fyi, this works to delay build for https://github.com/openshift/cucushift/pull/6453 . The delay can be seen in the logs:
$ oc logs -f --timestamps bc/build-delay
2018-06-20T08:25:37.416867675Z Cloning "https://github.com/xingxingxia/v3-testfiles.git" ...
2018-06-20T08:25:38.719786796Z 	Commit:	e1ccdb6764a551f2c4ef69856e87ec2f3a62c5aa (OCP-10674 build sleep)
2018-06-20T08:25:38.719813061Z 	Author:	Xingxing Xia <xxia@redhat.com>
2018-06-20T08:25:38.719816583Z 	Date:	Wed Jun 20 16:01:10 2018 +0800
2018-06-20T08:25:43.014378232Z ---> Installing application source ...
2018-06-20T08:25:43.01662563Z ---> Building your Ruby application from source ...
2018-06-20T08:25:43.016704375Z ---> Running 'bundle install --retry 2 --deployment --without development:test' ...
2018-06-20T08:27:45.463765924Z 10s delay time passed
2018-06-20T08:27:45.463801903Z 10s delay time passed
2018-06-20T08:27:46.229072859Z Fetching gem metadata from https://rubygems.org/..............
2018-06-20T08:27:46.451253242Z Fetching version metadata from https://rubygems.org/.
2018-06-20T08:27:46.627310254Z Installing puma 3.10.0 with native extensions
2018-06-20T08:27:48.172039788Z Installing rack 2.0.3
2018-06-20T08:27:48.234372057Z Using bundler 1.13.7
2018-06-20T08:27:48.242423927Z Bundle complete! 2 Gemfile dependencies, 3 gems now installed.
2018-06-20T08:27:48.242445305Z Gems in the groups development and test were not installed.
2018-06-20T08:27:48.242449071Z Bundled gems are installed into ./bundle.
2018-06-20T08:27:48.271315559Z ---> Cleaning up unused ruby gems ...
2018-06-20T08:27:48.456882564Z Running `bundle clean   --verbose` with bundler 1.13.7
2018-06-20T08:28:48.483716005Z 10s delay time passed
2018-06-20T08:28:48.48373282Z Found no changes, using resolution from the lockfile
2018-06-20T08:29:51.799867105Z 
2018-06-20T08:29:51.799899401Z Pushing image docker-registry.default.svc:5000/xxia-proj/build-delay:latest ...
2018-06-20T08:29:52.545001458Z Pushed 5/6 layers, 87% complete
2018-06-20T08:29:54.245462395Z Pushed 6/6 layers, 100% complete
2018-06-20T08:29:55.051224802Z Push successful

Reply:
This PR cloned https://github.com/openshift/ruby-ex, added the 2 lines and used it via --context-dir in oc new-app